sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -88,14 +90,14 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
      * @note See https://en.cppreference.com/w/cpp/chrono/duration.
      *
      */
-    std::chrono::milliseconds RetryDelay = std::chrono::seconds(4);
+    std::chrono::milliseconds RetryDelay = std::chrono::milliseconds(800);
 
     /**
      * @brief The maximum permissible delay between retry attempts.
      * @note See https://en.cppreference.com/w/cpp/chrono/duration.
      *
      */
-    std::chrono::milliseconds MaxRetryDelay = std::chrono::minutes(2);
+    std::chrono::milliseconds MaxRetryDelay = std::chrono::seconds(60);
```

Comment:
Nice, thanks!

These two changes are worth calling out in the breaking change section within the changelog (updated to be consistent across other language SDKs). The workaround/remediation for anyone who wants the old behavior is quite straight forward.
https://github.com/Azure/azure-sdk-for-net/blob/2b44628fd882aeaf77f6c59705fcb2c0b84f925f/sdk/core/Azure.Core/src/RetryOptions.cs#L53

https://github.com/Azure/azure-sdk-for-java/blob/76cc5783df8321cc2a63c8f3b5033081b597f203/sdk/core/azure-core/src/main/java/com/azure/core/http/policy/ExponentialBackoff.java#L26

Reply:
That makes sense. I'll update the changelog.

Reply:
@ahsonkhan @LarryOsterman How sure you are that this is worth being categorized as "Breaking Change"? I'm 50/50.

Reply:
Technically speaking, changing timeouts is a breaking change, especially if there's a server out there which is on the edge of compliance. And we're reducing the timeouts by a fair amount (2 minute to 1 minute max delay, 800ms per retry rather than 4s per retry).

Specifically, the 4s->800ms change might trigger other behaviors on a server since it's a fairly significant reduction in retry timeout (the change significantly affected the time for running our unit tests).

Reply:
Also, we are modifying a previously set default value, which is considered a bit breaking (regardless of context), or at the very least could be surprising change of behavior to an end user. Calling it out, to draw some attention to it/notify, just so customers know what happened seems like a good practice to me :)